Header header
float64 cellLength
float64 cellWidth
Obstacle[] obstacles
